pkg/kubectl/cmd/apply_test.go

```diff
@@ -279,6 +281,64 @@ func TestApplyObject(t *testing.T) {
 	}
 }
 
+func TestApplyObjectOutput(t *testing.T) {
+	initTestErrorHandler(t)
+	nameRC, currentRC := readAndAnnotateReplicationController(t, filenameRC)
+	pathRC := "/namespaces/test/replicationcontrollers/" + nameRC
+
+	// Add some extra data to the post-patch object
+	postPatchObj := &unstructured.Unstructured{}
+	if err := json.Unmarshal(currentRC, &postPatchObj.Object); err != nil {
+		t.Fatal(err)
+	}
+	postPatchLabels := postPatchObj.GetLabels()
+	if postPatchLabels == nil {
+		postPatchLabels = map[string]string{}
+	}
+	postPatchLabels["post-patch"] = "value"
```

Comment:
Here you are testing the case of `adding` a new label, how about update this case a bit by `updating` the label to reflect the real problem that want to fix?

Such as old label
```
name: oldLabel
```

New Label:
```
name: newLabel
```

Then the expect would be the label `name` was updated from `oldLabel` to `newLabel`.